Jenkinsfile

```diff
@@ -29,12 +29,26 @@ pipeline {
 
         stage("Test conjur_host_identity role") {
           steps {
-            sh './ci/test.sh -d conjur_host_identity'
-            junit 'roles/conjur_host_identity/tests/junit/*'
+            // sh './ci/test.sh -d conjur_host_identity'
+            // junit 'roles/conjur_host_identity/tests/junit/*'
+            sh 'chmod +x ./tests/ansibletest.sh'
```

Comment:
It looks like you've disabled the `conjur_host_identity` role's tests here. Running the `tests/ansibletest.sh` script should happen elsewhere in the Jenkins pipeline.

Reply:
Sure John,

Reply:
Done